.travis.yml

```diff
@@ -159,20 +162,26 @@ jobs:
       before_install: mkdir -p $HOME/.bin; export PATH=$PATH:$HOME/.bin; .travis/before_install.sh
       install: .travis/install.sh
       script:
-        - RAIDENVERSION=$TRAVIS_TAG make bundle-docker
+        - . .travis/set_tag.sh
```

Comment:
what is that first dot doing there? That looks funny

Reply:
It's sourcing the script. But good point using `source` is more explicit

Reply:
yeah thought it was equivalent to `source` but wasn't 100% sure.